REQUIRE

```diff
@@ -1,6 +1,6 @@
 julia 0.6.1
 Conda 0.5.3
-PyCall 1.10.0
+PyCall 1.10.0 1.18.6
```

Comment:
```suggestion
PyCall 1.10 1.19
```
No reason to limit at a specific `1.18.x` version

Reply:
I think the previous error
```julia
ERROR: LoadError: LoadError: LoadError: LoadError: ArgumentError: Module Libdl not found in current path.
Run `Pkg.add("Libdl")` to install the Libdl package.
```
came from using a more recent version of PyCall. But as you pointed out, the CI script was checking it out, so maybe this is indeed not necessary.